content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -312,6 +318,12 @@ In addition to the steps in the previous section, perform the following:
     Subject: C=US, ST=Denial, L=Springfield, O=Dis, CN=httpbin.example.com
     {{< /text >}}
 
+1.  If the secret was created but the keys were not mounted, kill the ingress gateway pod and force it to reload certs:
```

Comment:
This is redundant, since the Mutual TLS troubleshooting instructions already say "in addition to the above ..."

Reply:
@frankbu note that it is another certificate, loaded from another volume/secret. There could be a case when the certificate above is loaded successfully, while this certificate is not not, and killing the pod would be required again.

Reply:
I see. I guess it would be clearer if the second one said "the CA certificate secret", instead of just "the secret", same as the first one.